tests/autoyast/installation.pm

```diff
@@ -128,7 +128,7 @@ sub run {
     push @needles, 'autoyast-error' unless get_var('AUTOYAST_EXPECT_ERRORS');
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
```

Comment:
THanks, could you please add comment here why automatic reboot without confirmation is disappear for sle15sp7?, do we have any jira ticket/bug report or some user requirements?

btw, howabout s390x? or other arrches, and other hypervisors :)

Reply:
> THanks, could you please add comment here why automatic reboot without confirmation is disappeared for sle15sp7?, do we have any jira ticket/bug report or some user requirements?
> 
> btw, how about s390x? or other arrches, and other hypervisors :) IMO they may get the popup windows as well

Reply:
Thanks for review. In fact, the fix for https://bugzilla.suse.com/show_bug.cgi?id=1231522 add 'reboot' popup after installation on powerVM, user need click ok to reboot. For s390x, on latest build with the fix, it still be automatic reboot. For backport, developer is asking whether need backport to 15SP5 and 15SP6 in bug.
Comments added.